test/data/notificationExample.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I think this is a database entry right? At first I interpreted this as a notification response. Maybe rename to something like "notificationDatabaseEntry.json"?